src/scrabble.js

```diff
@@ -1,6 +1,48 @@
+const val1 = ["A", "E","I","O","U","L","N","R","S","T","a","e","i","o","u","l","n","r","s","t",];
+const val2 = ["D", "d", "G", "g"];
+const val3 = ["B", "b", "C", "c", "M", "m", "P", "p"];
+const val4 = ["F", "f", "H", "h", "V", "v", "W", "w", "Y", "y"];
+const val5 = ["K", "k"];
+const val8 = ["J", "j", "X", "x"];
+const val10 = ["Q", "q", "Z", "z"];
+
```

Comment:
A better way to do this, which would make it more maintainable, would be to put both the letters and corresponding values as key-value pairs in an object, e.g.
const scoresObject = { A: 1, E: 1, ...etc }